application/workflows/auto_novel_generation_workflow.py

```
@@ -177,20 +179,35 @@ async def post_process_generated_chapter(
+        consistency_report = self._review_chapter_seam(
+            novel_id=novel_id,
+            chapter_number=chapter_number,
+            content=content,
+            base_report=consistency_report,
+            rewrite_info=seam_rewrite_info,
+        )
         ghost_annotations = self._detect_conflicts(novel_id, chapter_number, outline, scene_director)
         if self.state_updater:
             try:
                 self.state_updater.update_from_chapter(novel_id, chapter_number, chapter_state)
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)